File: swagger/models/get_staff_permissions_response.py
from .base_model import BaseModel

from .staff_permission_group import StaffPermissionGroup

class GetStaffPermissionsResponse(BaseModel):
	"""
	no description available

	Attributes:

		user_group
		Contains information about the requested staff member’s permission
		group.
	"""

	input_map = {
		'UserGroup': ('user_group', StaffPermissionGroup),
		}

	output_map = {
		'user_group': 'UserGroup',
		}

	user_group = None

File: swagger/Models/GetStaffPermissionsResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetStaffPermissionsResponse extends BaseModel
{
	/**
	 * Contains information about the requested staff member’s permission
	 * group.
	 * 
	 * @var StaffPermissionGroup 
	 */
	public $userGroup;

	protected function getInputMap()
	{
		return [
			'UserGroup' => ['userGroup', StaffPermissionGroup::class, null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'userGroup' => ['UserGroup', StaffPermissionGroup::class, null],
			];
	}
}
